delta/app/src/main/resources/app.conf

```diff
@@ -244,7 +234,7 @@ app {
       snapshot-strategy = ${app.defaults.snapshot-default}
       stop-strategy = ${app.defaults.passivate-never}
       ask-timeout = 15 seconds
-      evaluation-max-duration = 3 seconds
+      evaluation-max-duration = 11 seconds
```

Comment:
Why not 15s ?

Reply:
it must be smaller than the `ask-timeout`. But yes, It can be 14s for example